README.md

```diff
@@ -1,152 +1,119 @@
 
 ## 🛠 Built With <a name="built-with"></a>
-1.HTML
-2.CSS
-3.Java-Script
-4.Linters
-### Tech Stack <a name="tech-stack"></a>
-### Key Features <a name="key-features"></a>
-Add index.html file 
-Add style.css file
-Add Java-Script file
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> coming soon
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-## 💻 Getting Started <a name="getting-started"></a>
-
-### Prerequisites
-
-In order to run this project you need:
- .Web browser
- .Code editor 
- .Git-smc 
-
-### Setup
-
-Clone this repository to your desired folder:
-Run this command:
-```sh
-cd my-Folder
-git clone git@github.com:Nessrine88/Capstone-Project.git
-```
-### Install
-
-Install this project with:
-
-
-```sh
-  cd my-project
-  npm install
-```
-### Usage
-
-To run the project, execute the following command:
-open index.html using live server extension
-
-
-### Run tests
-
-coming soon
-
 
-### Deployment
+The project is built using the following technologies and tools:
 
-https://nessrine88.github.io/Portfolio.github.io/
+- HTML
+- CSS
+- JavaScript
 
+## Key Features <a name="key-features"></a>
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+The Conference Website offers the following key features:
 
+The Prickly Pear Seed Oils Company website offers the following key features:
 
-## 👥 Authors <a name="authors"></a>
+-**Product Showcase**: Showcases different types of Prickly Pear Seed Oils with detailed descriptions, product images, and pricing information.
+-**Benefits and Uses**: Provides information about the benefits and uses of Prickly Pear Seed Oils, highlighting their skincare, health, and culinary applications.
+-**Online Store**: Offers a secure and user-friendly online store where visitors can add products to their cart, view their cart, and complete the purchase process.
+-**Customer Reviews**: Displays customer reviews and testimonials to build trust and credibility for the products.
+-**About Us**: Provides background information about the company, its values, and its commitment to producing high-quality Prickly Pear Seed Oils.
+-**Contact Us**: Includes contact information and a contact form for inquiries and customer support.
+We hope this clarifies the nature of the Capstone Project as a Prickly Pear Seed Oils Company website.
 
+## Acknowledgements <a name="acknowledgements"></a>
 
+We would like to express our gratitude to Cindy Shin, the author of the original design used for this project. Her Creative Commons License allows us to adapt and showcase her design in this capstone project. We appreciate her contribution to the design community.
 
```

Comment:
- [ Required ] Good job adding attribution to Cindy Shin, the original author of this design. However, you forgot to include the links to her profile and design. I would like to suggest that you update this section with the following text so that your attribution is consistent with the [Creative Commons License](https://creativecommons.org/licenses/).

## Acknowledgments
We would like to express our gratitude to [Cindy Shin](https://www.behance.net/adagio07)[Cindy Shin](https://www.behance.net/adagio07), [the author of the original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) used for this project. Her [Creative Commons License](https://creativecommons.org/licenses/) allows us to adapt and showcase her design in this capstone project. We appreciate her contribution to the design community.